src/mlpack/tests/CMakeLists.txt

```diff
@@ -90,11 +70,17 @@ add_executable(mlpack_catch_test
   layer_names_test.cpp
   lin_alg_test.cpp
   linear_regression_test.cpp
-  lmnn_test.cpp
```

Comment:
why removed ?

Reply:
Hey looks like this was something that happened during the merge conflict, this was a mistake I have added it back.

Reply:
Jeffin, Thanks for catching this and making my review much easier :)